src/redux/actions/userActions.js

```diff
@@ -5,3 +5,8 @@ export const signInUserSuccess = createAction('SIGN_IN_USER_SUCCESS');
 export const signInUserError = createAction('SIGN_IN_USER_ERROR');
 export const signIn = credentials =>
   createDispatcher(userApi.signIn, signInUserSuccess, signInUserError, credentials);
+
+export const signUpSuccess = createAction('SIGN_UP_USER_SUCCESS');
+export const signUpError = createAction('SIGN_UP_USER_ERROR');
+export const signUp = params =>
```

Comment:
You are gonna love I really like this abstraction!
I think you are gonna love [redux-tools](https://github.com/rootstrap/redux-tools)